kvirt/k3s/bootstrap.sh

```diff
@@ -8,8 +8,7 @@ export IP={{ api_ip if masters > 1 else '$(hostname -I | cut -f1 -d" ")' }}
 export K3S_TOKEN=$(cat /var/lib/rancher/k3s/server/node-token)
 sed "s/127.0.0.1/$IP/" /etc/rancher/k3s/k3s.yaml > /root/kubeconfig
 if [ -d /root/manifests ] ; then
- mkdir -p {{ data_dir }}
- mv /root/manifests {{ data_dir|default("/var/lib/rancher/k3s/server") }}
+    mv /root/manifests {{ data_dir|default("/var/lib/rancher/k3s/server") }}
```

Comment:
when you deploy, this directory wont exist

Reply:
I believe so it will. As when K3s is up and running - which it should be at this point in time - the folder structure that K3s requires will have been created by K3s itself.
That's what I've seen so far. Hence my thinking here that `mkdir ...` is not needed.